kernel/aster-nix/src/fs/mod.rs

```diff
@@ -64,3 +65,26 @@ pub fn lazy_init() {
         self::rootfs::mount_fs_at(exfat_fs, &target_path).unwrap();
     }
 }
+
+pub fn get_fs(fs_type: &str) -> Result<Arc<dyn FileSystem>> {
+    match fs_type {
+        "ext2" => {
+            if let Ok(block_device_ext2) = start_block_device("vext2") {
+                let ext2_fs = Ext2::open(block_device_ext2).unwrap();
```

Comment:
Each time this function is called, a new kernel thread is spawned to handle requests for the underlying device. This behavior is undesirable. Instead, we should modify the function to accept an already started block device as an argument.